program/src/processor.rs

```diff
@@ -335,29 +335,23 @@ fn process_vote(
 
     let clock = <Clock as Sysvar>::get()?;
 
-    // Ensure the voting period has not ended.
-    if proposal_state.voting_has_ended(governance_config.voting_period_seconds, &clock) {
-        // If it has, determine whether the proposal was accepted or rejected
-        // by checking if it has a cooldown. If it does not, the proposal is
-        // rejected.
-        // Proposals with cooldowns when voting ends are automatically
-        // accepted, regardless of cooldown time remaining.
-        if proposal_state.cooldown_timestamp.is_some() {
-            proposal_state.cooldown_timestamp = None;
-            proposal_state.status = ProposalStatus::Accepted;
-        } else {
-            proposal_state.status = ProposalStatus::Rejected;
-        }
+    // If the proposal has an active cooldown period, ensure it has not ended.
+    if proposal_state.cooldown_has_ended(governance_config.cooldown_period_seconds, &clock) {
+        // If the cooldown period has ended, the proposal is accepted.
+        proposal_state.cooldown_timestamp = None;
```

Comment:
Is there a specific reason to clear this field? It seems like it could be useful to know when the voting finally ended

Reply:
Yeah, I can keep this, doesn't really matter. I just figured since it's the beginning timestamp of cooldown, it wouldn't really be interesting to anyone, but I could be wrong!

Reply:
It can tell us exactly when the proposal ended, which might be useful for UIs